manifests/openshift-templates/list.yaml

```diff
@@ -1,491 +1,4304 @@
+                      - MAP<INT, INT>
+                      - MAP<INT, VARCHAR>
+                      - ROW
+                      - IPADDRESS
+                      - UUID
+                      - HYPERLOGLOG
+                      - P4HYPERLOGLOG
+                      - QDIGEST
+                      - boolean
+                      - tinyint
+                      - smallint
+                      - integer
+                      - bigint
+                      - real
+                      - double
+                      - decimal
+                      - varchar
+                      - char
+                      - varbinary
+                      - json
+                      - date
+                      - time
+                      - timestamp
+                      - array
+                      - map
+                      - map<varchar, varchar>
+                      - map<varchar, int>
+                      - map<int, int>
+                      - map<int, varchar>
+                      - row
+                      - ipaddress
+                      - uuid
+                      - hyperloglog
+                      - p4hyperloglog
+                      - qdigest
+                  unit:
+                    type: string
+                    enum:
+                      - date
+                      - kubernetes_pod
+                      - kubernetes_persistentvolumeclaim
+                      - kubernetes_persistentvolume
+                      - kubernetes_storageclass
+                      - kubernetes_namespace
+                      - kubernetes_node
+                      - bytes
+                      - byte_seconds
+                      - time
+                      - cpu_core_seconds
+                      - cpu_cores
+                      - memory_bytes
+                      - memory_byte_seconds
+                      - seconds
+                  tableHidden:
+                    type: boolean
+            inputs:
+              type: array
+              minItems: 1
+              items:
+                type: object
+                required:
+                - name
+                properties:
+                  name:
+                    type: string
+                    minLength: 1
+                  type:
+                    type: string
+                    enum:
+                      - string
+                      - integer
+                      - time
+                      - ReportDataSource
+                      - ReportQuery
+                      - Report
+                  required:
+                    type: boolean
+                  default: {}
+            query:
+              type: string
+              pattern: '[Ss][Ee][Ll][Ee][Cc][Tt]\s'
+              minLength: 1
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: storagelocations.metering.openshift.io
+spec:
+  group: metering.openshift.io
+  version: v1
+  versio
```

Comment:
Why did you delete the template? We need everything in here to be in the openshift template, that's how saasherder works (https://github.com/openshiftio/saasherder#the-process). Also, everything in here needs to be a list item under `objects` in the Openshift template.